README.md

```diff
@@ -95,19 +95,12 @@ You can either call modules by their Fully Qualified Collection Name (FQCN), suc
 ## Contributing to this collection
 
 We welcome community contributions to this collection. If you find problems, please open an issue or create a PR against the [Amazon AWS collection repository](https://github.com/ansible-collections/amazon.aws).
-See [Contributing to Ansible-maintained collections](https://docs.ansible.com/ansible/devel/community/contributing_maintained_collections.html#contributing-maintained-collections) for more details.
+See [CONTRIBUTING.md](CONTRIBUTING.md) for more details.
```

Comment:
I'm not sure if this would cause problems with Galaxy.  We might be better adding CONTRIBUTING.md to extradocs (see #1047) and then linking to the GitHub Pages docs.

Reply:
I don't understand what extradocs is. What do you want me to do?

Do you mean something like this?

```
See the [rendered changelog](https://ansible-collections.github.io/amazon.aws/branch/main/collections/amazon/aws/docsite/CHANGELOG.html) or the [raw generated changelog](https://github.com/ansible-collections/amazon.aws/tree/main/CHANGELOG.rst).
```

Reply:
[extra-docs.yml](https://github.com/ansible-collections/amazon.aws/blob/main/docs/docsite/extra-docs.yml)  adds extra links to the sidebar of the renders docs, and towards the top of the main [index.html](https://ansible-collections.github.io/amazon.aws/branch/main/collections/amazon/aws/index.html).

If you look in docs/docsite/rst you'll see there's a symlink to CHANGELOG.rst
```
$ readlink CHANGELOG.rst 
../../../CHANGELOG.rst
```

The entry in extra-docs.yml then adds this link
```
  - title: Changelog
    toctree:
      - CHANGELOG
```

Once that's in place we can link to it in README.md as https://ansible-collections.github.io/amazon.aws/branch/main/collections/amazon/aws/docsite/CONTRIBUTING.html as was done with the README.md change in #1047.   In a similar vein to - https://github.com/ansible-collections/amazon.aws/pull/1047/commits/a0e7fc0332ee1b429e9815c9ab90f6c819386a81